### ISSUE TITLE
iframe problem

### ISSUE THREAD

**Issue by AndySilvia** [nesges/Widgets-for-fhem-tablet-ui#8]
Problem is, widget_iframe should be under widget_famultibutton - it's not correct in your hierarchy-view too...

doesn't do anything, but errors, you have to add these lines at the beginning of widget_iframe.js:

function depends_iframe (){
    if(typeof Module_famultibutton == 'undefined')
        return ["famultibutton"];
};

Bye & thanks for the good work!

**Comment by nesges** [comment]
No. Most certainly you are using a Version of ftui that isn't compatible
with my Widgets anymore. ftui  has fundamentaly changed in the last version.

2016-10-11 1:07 GMT+02:00 AndySilvia notifications@github.com:

**Comment by AndySilvia** [comment]
Ok. Thanks for clarification. Anyway the widget does what it should with the modification I added...

**Comment by Memphiz** [comment]
@nesges any plans to adapt your widgets for the newest ftui version? I am interested in the klimatrend widget which doesn't work currently (loadplugin not defined ...)
